samza-kv/src/main/scala/org/apache/samza/storage/kv/LoggedStore.scala

```diff
@@ -114,4 +114,7 @@ class LoggedStore[K, V](
     store.close
   }
 
+  override def iterate(from: K, to: K): KeyValueIterable[K, V] = {
+    store.iterate(from, to)
```

Comment:
where this iterate comes from?

Reply:
I added it to the general interface of KeyValueIterable.